Image pull policy (#1345)
testcontainers/testcontainers-java: docs/features/advanced_options.md

[Adding multiple labels](../examples/src/test/java/generic/ContainerLabelTest.java) inside_block:multiple_labels
<!--/codeinclude-->

## Image Pull Policy

By default, the container image is retrieved from the local Docker images cache.
This works well when running against a specific version, but for images with a static tag (i.e. 'latest') this may lead to a newer version not being pulled.

> It is possible to specify an Image Pull Policy to determine in runtime whether an image should be pulled, either by using one of the existing policies:

Reviewer: ```suggestion
It is possible to specify an Image Pull Policy to determine at runtime whether an image should be pulled or not:
```
